src/plays/navbar/NavBar.jsx

```diff
@@ -0,0 +1,65 @@
+import { getPlayById } from 'meta/play-meta-util';
+
+import PlayHeader from 'common/playlists/PlayHeader';
+import './navBar.css';
+import Menus from "./Navbar-image"
+import { useState } from 'react';
+
+function NavBar(props) {
+  // Do not remove the below lines. 
+  // The following code is to fetch the current play from the URL
+  const { id } = props;
+  const play = getPlayById(id);
+
+  // Your Code Start below.
+  const [open, setOpen] = useState(false);
+
+  return (
+    <>
+      <div className="play-details">
+        <PlayHeader play={play} />
+        <div className="play-details-body">
+          {/* Your Code Starts Here */}
+          <div className='flex'>
+
+            <div className={`${open ? 'w-72' : 'w-20'} 
+            duration-500 h-screen p-5 pt-8 bg-slate-900 relative`}>
+
+              <img src={require('../navbar/assest/img.png')}
+                alt=''
+                className={`absolute cursor-pointer rounded-full -right-3 top-9 w-7 border-2 bg-cyan-500 ${!open && "rotate-180"}`}
+                onClick={() => setOpen(!open)} />
+
+              <div className='flex gap-x-4 items-center'>
+                <img src={require('../navbar/assest/logo.png')}
+                  alt=''
```

Comment:
Can we add some value to the alt attribute?

```suggestion
                  alt='play-logo'
```